invenio_records_files/api.py

```diff
@@ -112,34 +158,59 @@ def __init__(self, record, bucket=None, file_cls=None):
 
     @property
     def keys(self):
-        """Return file keys."""
+        """Return file keys.
+
+        :returns: Key list of the files files.
```

Comment:
@JavierDelgadoFernandez can you rephrase this line? `List of file identifiers/names`?